lib/screens/forgot_password/forgot_password_repository.dart

```diff
@@ -3,42 +3,38 @@ import 'dart:math';
 import 'package:ecellapp/core/res/errors.dart';
 
 abstract class ForgotPasswordRepository {
-  Future<String> sendOTP(String email);
+  Future<void> sendOTP(String email);
 
-  Future<String> verifyOTP(String otp);
+  Future<void> verifyOTP(String otp);
 
-  Future<String> changePassword(String email, String otp, String password);
+  Future<void> changePassword(String email, String otp, String password);
 }
 
 class FakeForgotPasswordRepository extends ForgotPasswordRepository {
   @override
   // this is to simulate a delay for getting otp
-  Future<String> sendOTP(String email) async {
+  Future<void> sendOTP(String email) async {
     await Future.delayed(Duration(seconds: 2));
     if (Random().nextBool()) {
       return "200";
```

Comment:
no need to return anything when return type is void.
```suggestion
      return;
```